### PR TITLE
Fix settings export

### LINKED ISSUE #9789: Files carnt export settings
### Description

When exporting the settings Files make a blank corrupted zip file.

### Steps To Reproduce

_No response_

### Expected behavior

Settings should be exported

### Files Version

2.3.9.0

### Windows Version

10.0.19043.1889

### Relevant Assets/Logs

[debug.log](https://github.com/files-community/Files/files/9390584/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/9390585/debug_fulltrust.log)

### LINKED ISSUE #9799: Crash when opening properties of a ADS file
### Description

*

### Steps To Reproduce

_No response_

### Expected behavior

The button should be disabled or redirect to the actual main file

### Files Version

2.3.9.0

### Windows Version

10.0.19043.1889

### Relevant Assets/Logs

[debug.log](https://github.com/files-community/Files/files/9396464/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/9396465/debug_fulltrust.log)

### PR DESCRIPTION
**Pull request by gave92**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #9789
- Closes #9799

**Details of Changes**
Add details of changes here.
- Code for export needed to be updated for the new 7z library
- Fix crash when viewing properties of file without extension

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

File: src/Files.Uwp/Helpers/FileExtensionHelpers.cs [hunk 1]
```diff
@@ -48,12 +48,23 @@ public static bool IsPowerShellFile(string fileExtensionToCheck)
...
+
             return new[] { ".zip", ".msix", ".appx", ".msixbundle", ".7z", ".rar", ".tar" }
                 .Contains(fileExtensionToCheck, StringComparer.OrdinalIgnoreCase);
         }
 
         public static bool IsBrowsableZipFile(string filePath, out string ext)
         {
+            if (string.IsNullOrEmpty(filePath))
+            {
+                ext = null;
```
Comment (d2dyno1) [inline src/Files.Uwp/Helpers/FileExtensionHelpers.cs:64]: Can this be null? (you could always use string.Empty)

**Review by d2dyno1 — APPROVED** [review]
Could you also add these changes to `winui3_mini` later? 🙂

File: src/Files.Uwp/Helpers/FileExtensionHelpers.cs (hunk 1 above)
Comment (gave92) [reply src/Files.Uwp/Helpers/FileExtensionHelpers.cs:64]: If the function returns false, you know that the `out` parameter will be invalid. The actual value of `ext` in that case does not matter.

**Comment by gave92** [comment]
> Could you also add these changes to `winui3_mini` later? 🙂 

Sure thing 👍

**Pull request merged**